wallet/components/ShieldedMode.tsx

```diff
@@ -90,28 +100,38 @@ function TabGroup(p: { children: ReactNode }) {
         </Horizontal>
       </div>
       <div>{content}</div>
+      <div>Shielded: {isShielded.toString()}</div>
+      <div>Virtual: {isVirtual.toString()}</div>
     </div>
   );
 }
 
-export function ShieldedTabs(p: { public: ReactNode; private: ReactNode }) {
-  const { isShielded, setShielded } = useShieldedApi();
+export function ShieldedTabs(p: { public: ReactNode; private: ReactNode; virtual: ReactNode }) {
+  const { isShielded, setShielded, isVirtual, setVirtual } = useProfileTabApi();
+
   return (
     <TabGroup>
       <Tab
-        active={!isShielded}
+        active={!isShielded && !isVirtual}
         title="Public"
-        onClick={() => setShielded(false)}
+        onClick={() => {setShielded(false); setVirtual(false)}}
```

Comment:
And when you click on it, it would be something like `setTab(Tab.PUBLIC)`

Reply:
Sound!

Reply:
Done!